app/models/user.rb

```diff
@@ -7,6 +7,8 @@ class User < ApplicationRecord
   accepts_nested_attributes_for :portfolio
   has_many :orders, dependent: :destroy
 
+  after_create :create_portfolio_for_student
```

Comment:
We have this method on [Student](https://github.com/rubyforgood/stocks-in-the-future/blob/main/app/models/student.rb#L6).  Do you think it should be here instead of on Student?